tests/conftest.py

```diff
@@ -9,3 +11,13 @@ def mock_local_url(monkeypatch):
 @pytest.fixture
 def mock_dev_url(monkeypatch):
     monkeypatch.setenv("DEV_URL", "http://docker-dev03.pcic.uvic.ca/somebird")
+
+
+@pytest.fixture
+def wps_test_collect_args(monkeypatch):
+    return TestCollectArgs()
+
+
+@pytest.fixture
+def wps_test_process(monkeypatch):
+    return TestProcess()
```

Comment:
I think it would be preferable to have all the class definitions in here rather than importing them.